test/test-bcf-sr.c

```diff
@@ -133,13 +133,15 @@ int main(int argc, char *argv[])
         {"targets",required_argument,NULL,'t'},
         {"no-index",no_argument,NULL,1000},
         {"args",no_argument,NULL,1001},
+        {"usefptr",no_argument,NULL,'u'},
```

Comment:
Please add a line to the usage statement too.
Thanks

Reply:
updated.